projects/distributed-tracing/src/shared/components/log-events/log-events-table.component.ts

```diff
@@ -93,29 +91,58 @@ export class LogEventsTableComponent implements OnChanges {
   }
 
   private getTableColumnConfigs(): TableColumnConfig[] {
-    if (this.logEventsTableViewType === LogEventsTableViewType.Condensed) {
-      return [
-        {
-          id: 'timestamp',
-          name: 'timestamp',
-          title: 'Timestamp',
-          display: CoreTableCellRendererType.RelativeTimestamp,
-          visible: true,
-          width: '150px',
-          sortable: false,
-          filterable: false
-        },
-        {
-          id: 'summary',
-          name: 'summary',
-          title: 'Summary',
-          visible: true,
-          sortable: false,
-          filterable: false
-        }
-      ];
+    switch (this.logEventsTableViewType) {
+      case LogEventsTableViewType.Condensed:
+        return [
+          {
+            id: 'timestamp',
+            name: 'timestamp',
+            title: 'Timestamp',
+            display: CoreTableCellRendererType.RelativeTimestamp,
+            visible: true,
+            width: '150px',
+            sortable: false,
+            filterable: false
+          },
+          {
+            id: 'summary',
+            name: 'summary',
+            title: 'Summary',
+            visible: true,
+            sortable: false,
+            filterable: false
+          }
+        ];
+      case LogEventsTableViewType.Detailed:
+        return [
+          {
+            id: 'timestamp',
+            title: 'Timestamp',
+            display: CoreTableCellRendererType.RelativeTimestamp,
+            visible: true,
+            width: '150px',
+            sortable: false,
+            filterable: false
+          },
+          {
+            id: 'summary',
+            title: 'Summary',
+            visible: true,
+            sortable: false,
+            filterable: false
+          },
+          {
+            id: '$$spanName',
+            title: 'Span',
+            display: WaterfallTableCellType.SpanName,
```

Comment:
should this column be navigable?

Reply:
Actually, there is no requirement for this. This is for view only I guess.